libretro-common/file/config_file.c

```diff
@@ -466,40 +466,48 @@ static bool config_file_parse_line(config_file_t *conf,
       char real_path[PATH_MAX_LENGTH];
       char *path               = NULL;
       char *include_line       = NULL;
+      char *reference_line     = NULL;
 
       /* Starting a line with an 'include' directive
        * appends a sub-config file
        * > All other comments are ignored */
       if (!string_starts_with_size(comment, "include ",
-               STRLEN_CONST("include ")))
+               STRLEN_CONST("include ")) &&
+          !string_starts_with_size(comment, "reference ",
+               STRLEN_CONST("reference ")))
```

Comment:
You end up doing these `string_starts_with_size()` calls twice (potentially). This is a bit slow. The return values could be cached and reused.